File: L9/Java/15.js
let frase = prompt("Digite uma frase:");
let numBrancos = 0;

for (let i = 0; i < frase.length; i++) {
  if (frase[i] === ' ') {
    numBrancos++;
  }
}

console.log(`Número de caracteres em branco: ${numBrancos}`);

File: L9/Python/15.py
frase = input("Digite uma frase: ")

quantidade_caracteres_brancos = 0

for caractere in frase:
    if caractere == ' ':
        quantidade_caracteres_brancos += 1

print(f"A frase '{frase}' possui {quantidade_caracteres_brancos} caracteres em branco.")
